examples/c/Makefile

```diff
@@ -64,6 +75,14 @@ $(BPFTOOL): | $(dir $(BPFTOOL))
 	$(call msg,BPFTOOL,$@)
 	$(Q)$(MAKE) OUTPUT=$(dir $(BPFTOOL)) -C $(BPFTOOL_SRC)
 
+$(LIBBLAZESYM_OBJ):: $(LIBBLAZESYM_SRC)/src/*.rs $(LIBBLAZESYM_SRC)/Cargo.toml $(OUTPUT)
```

Comment:
I should remove *.rs but keep double colons.  The directory of blazesym is getting complicated (submodules).  It is unrealistic to maintain dependencies here.  I want to run cargo to check dependencies for me.

Does that make sense?

Reply:
I found it does not work as I expected.  I will change it.

Reply:
oh, right, forgot to ask if it's possible to basically always run cargo instead of guessing dependencies. Ideally cargo will know when nothing should be done and skip building stuff, right? So basically you add FORCE as depenency to always call cargo build. You'll probably need to be careful to not copy .a file if it didn't change (or at least preserve original modification time), otherwise we'll keep rebuilding profile app all the time, even if nothing changed.